tests/ci/docker-compose-direct-model.yaml

```diff
@@ -15,11 +15,16 @@
 version: '3.5'
 
 services:
-  avalon-direct-model-tests:
+  avalon-tests:
     image: avalon-shell:${ISOLATION_ID}
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
     working_dir: "/project/avalon/"
```

Comment:
Can you add back the volume mapping that was present for logs?

Reply:
Yes, I lost it in moving code here and there. Will add it.